CHANGELOG.md

```
@@ -1,5 +1,39 @@
 # CHANGELOG
 
+## 6.4.0 (Current)
+
+### Breaking Changes
+
+- **Simplified requestProducts API**: The `requestProducts` method now accepts direct parameters instead of a wrapper object (Fixes [#527](https://github.com/hyochan/flutter_inapp_purchase/issues/527))
+  ```dart
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)